src/pages/error/BadRequest.js

```diff
@@ -11,22 +11,22 @@ import { styles } from './styles/bad-request.styles'
 
 const BadRequest = () => {
   const { t } = useTranslation()
-  const windowSize = useBreakpoints()
+  const { isDesktop, isTablet, isMobile } = useBreakpoints()
 
-  const sizesTypography = {
-    desktop: { title: 'h2', description: 'subtitle1' },
-    tablet: { title: 'h3', description: 'subtitle1' },
-    mobile: { title: 'h4', description: 'subtitle2' }
-  }
+  const desktop = { title: 'h2', description: 'subtitle1' }
```

Comment:
it will be better if you rename it to desktopTypography

Reply:
done.